utilities.js

```diff
@@ -0,0 +1,24 @@
+const products = require('./src/data/products')
+const categories = require('./src/data/categories')
+module.exports = {
+    getProductsByCategoryId : (categoryId) => {
+        return products.products.filter(value => value.categoryId === categoryId)
+    },
+    getCategoryById: (id) => {
+        return categories.categories.filter(value => value.id === id)[0];
+    },
+    getProductById: (id) => {
+        return products.products.filter(value => value.id === id)[0];
+    },
+    notFound: (res) => {
+        res.statusCode = 404
+        res.end()
```

Comment:
If it's 404, Didn't you want to send a response - Why it is 404? 
e.g: `Id not found`